src/goe/offload/offload_transport.py

```diff
@@ -357,7 +377,10 @@ def log(msg, messages):
         return False
 
     if offload_source_table:
-        if ORACLE_TYPE_INTERVAL_YM in offload_source_table.data_types_in_use():
+        if (
+            ORACLE_TYPE_INTERVAL_YM in offload_source_table.data_types_in_use()
+            and 1 == 2
```

Comment:
Is this `1 == 2` deliberate?

Reply:
No it isn't! How did I miss that! Thanks

Reply:
I've removed this code.